plugins/inventory/k8s.py

```diff
@@ -14,8 +14,9 @@
     short_description: Kubernetes (K8s) inventory source
 
     description:
-      - Fetch containers and services for one or more clusters
-      - Groups by cluster name, namespace, namespace_services, namespace_pods, and labels
+      - Fetch containers and services for one or more clusters.
+      - Groups by cluster name, namespace, namespace_services, namespace_pods, and labels.
+      - Uses the kubectl connection plugin to access the Kubernetes cluster.
```

Comment:
Is this line relevant for the end-user to know, or is it just an internal implementation detail?

Reply:
I thought it might be relevant to the end user. The connection plugin is the only plugin that requires `kubectl` rather than use the API directly. So I thought it was worth a mention in case the run into a problem related to this.